mod/admin.php

```diff
@@ -290,7 +290,7 @@ function admin_page_federation(&$a) {
 		// what versions for that platform do we know at all?
 		// again only the active nodes
 		$v = qu('SELECT COUNT(*) AS `total`, `version` FROM `gserver`
-				WHERE `last_contact` > `last_failure` AND `platform` LIKE "%s"  AND `version` != ""
+				WHERE `last_contact` > `last_failure` AND `platform` LIKE "%s"  AND `version` RLIKE "^[[:digit:]]+[-.][[:digit:]]+"
```

Comment:
We have no clue, how a version number looks like. Most of them starts with numbers. Nor does a type for version numbers exists. They are only strings. With this predictions I think, we should store what we get and build our own reports. A similar fix exists in the same function for Diaspora nodes. I'm don't happy too. But if we not store what we get we miss some new numbers.